build.gradle

```diff
@@ -39,7 +39,7 @@ if (project.hasProperty('overrideBuildEnvironment')) {
 }
 
 def avroVersion = '1.10.2'
-def avroUtilVersion = '0.2.150'
+def avroUtilVersion = '0.3.17'
```

Comment:
That looks like a big bump! We should look carefully to get confidence that there are no regressions....

Reply:
The CI passed, and we will certainly bake this in EI.
Fast-Avro hasn't updated much, and mostly the avro compatiblity layer.

Reply:
It's not really too big of a jump in production, actually. Our internal dependencies were bumping up `avro-util` transitively to a much more recent version (`0.3.14`)

Reply:
There is one regression related to `RandomRecordGenerator` that we've identified - [This PR](https://github.com/linkedin/avro-util/pull/495/files) breaks random object generation for non-record schemas